lib/ansible/modules/files/acl.py

```diff
@@ -27,12 +27,13 @@
 description:
     - Sets and retrieves file ACL information.
 options:
-  name:
+  path:
     required: true
     default: null
     description:
       - The full path of the file or object.
```

Comment:
description should have something like: Before 2.3 this option was only usable as 'name'